### ISSUE TITLE
Document our availability queries used in the Alma Api

### ISSUE THREAD

**Issue by tclayton33** [emory-libraries/blacklight-catalog#1404]
Please pull out the commands we are using in our availability queries from the Alma API (put together the full URL we are using for these queries). We want to pass these on to Ex Libris.
 
Background: This relates to the Linked Data focus group project and the fact that our availability and physical location information can't be pulled with the Retrieve Bib Api (the MARC inventory field cannot be added for BIBFRAME records). Ex Libris has suggested we use a different solution that involves using the SRU; however, their documentation for [configuring the SRU](https://developers.exlibrisgroup.com/alma/integrations/sru/) is a little light. We are hoping that if we pass our code to them, they can help us with the appropriate CQL replacements.

**Comment by abelemlih** [comment]
@tclayton33 @rotated8 here are the API GET requests we current send to ALMA to retrieve availability data:

```
Availability by Facet
https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs?mms_id=9937232320402486%2C9936966798002486%2C9937241319602486%2C9936997911202486%2C9936908988502486%2C9936904054102486%2C9937010522202486%2C9936972870302486%2C9936747008502486%2C9936710411802486&view=full&expand=p_avail,e_avail,d_avail&apikey=
```

```
Availability for Search Results
https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs?mms_id=990017323500302486%2C990007006050302486%2C990030289410302486%2C990023781120302486%2C990016154420302486%2C990007006030302486%2C990001359750302486%2C990025514480302486%2C990016154430302486%2C990024937200302486&view=full&expand=p_avail,e_avail,d_avail&apikey=
```

```
Availability for Single Item
https://api-na.hosted.exlibrisgroup.com/almaws/v1/bibs?mms_id=990017323500302486&view=full&expand=p_avail,e_avail,d_avail&apikey=
```

Only api keys are removed from the URLs above. I used my local environment to run the queries above, and my local has the collection `blacklightbrad` indexed from the ALMA Sandbox. In the catalog application, we currently rely on the `AlmaAvailabilityService` to retrieve this data.

**Comment by rotated8** [comment]
From ExL, solution for multiple MMSIDs: https://na01.alma.exlibrisgroup.com/view/sru/TR_INTEGRATION_INST?version=1.2&operation=searchRetrieve&recordSchema=marcxml&query=mms_id=992362980000561%20or%20mms_id=9980409300561&maximumRecords=3

**Comment by abelemlih** [comment]
SRU querying was implemented in #1415 and deployed to Arch under v1.14.0. @tclayton33 @rotated8 this ticket is ready to close.
